pkg/detectors/hubspotapikeyv2/hubspotapikeyv2.go

```diff
@@ -0,0 +1,76 @@
+package hubspotapikeyv2
+
+import (
+	"context"
+	// "log"
+	regexp "github.com/wasilibs/go-re2"
+	"net/http"
+	"strings"
+	"fmt"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/common"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/pb/detectorspb"
+)
+
+type Scanner struct{}
+
+// Ensure the Scanner satisfies the interface at compile time.
+var _ detectors.Detector = (*Scanner)(nil)
+
+var (
+	client = common.SaneHttpClient()
+
+	keyPat = regexp.MustCompile(`\b(pat-na1-[A-Za-z0-9]{8}\-[A-Za-z0-9]{4}\-[A-Za-z0-9]{4}\-[A-Za-z0-9]{4}\-[A-Za-z0-9]{12})\b`)
```

Comment:
```
pat-na1-
```
I can't find official documentation for this, but it looks like there are region-specific prefixes including `pat-na1-` and `pat-eu1-`. It's possible there are others.

Reply:
how did you find the pat-eu1 format? We can add these 2 formats for now and add more later?

Reply:
any updates on this?